update-submodule/action.yml

```diff
@@ -65,6 +65,13 @@ inputs:
     description: 'The message for the commit'
     default: 'Update submodule'
     required: false
+outputs:
+  sha:
+    description: "Full SHA1 of resulting commit"
```

Comment:
Please use single quotes for consistency with other parameters.

Reply:
fixed